interpretation/deepseismic_interpretation/dutchf3/data.py

```diff
@@ -463,6 +466,13 @@ def __getitem__(self, index):
             augmented_dict = self.augmentations(image=im, mask=lbl)
             im, lbl = augmented_dict["image"], augmented_dict["mask"]
 
+        # dump images and labels to disk
+        if self.debug:
+            outdir = f"patchLoader_{self.split}_{'aug' if self.augmentations is not None else 'noaug'}"
+            generate_path(outdir)
+            mask_to_disk(im, f"{outdir}/{index}_img.png")
+            mask_to_disk(lbl, f"{outdir}/{index}_lbl.png")
```

Comment:
See comment on mask_to_disk above. I think we need another function like "img_to_disk" to avoid having the color map applied to the seismic images as well.

Reply:
good feedback! Changing!